README.rst

```diff
@@ -168,6 +168,41 @@ Remove a container::
 
     courseraresearchexports containers remove $CONTAINER_NAME_OR_ID
 
+db
+^^
+
+create_view
+~~~~~~~~~~~
+Create a view in the dockerized database. We are planning to include commonly
+used denormalized views as part of this project. To create one of these views
+(i.e. for the demographic_survey view)::
+
+    courseraresearchexports db create_view $CONTAINER_NAME_OR_ID --view_name demographic_survey --partner_short_name $PARTNER_SHORT_NAME
+
+If you have your own sql that you'd like to create as a view::
+
+    courseraresearchexports db create_view $CONTAINER_NAME_OR_ID --sql_file /path/to/sql/file/ --partner_short_name $PARTNER_SHORT_NAME
+
+unload_to_csv
+~~~~~~~~~~~~~
+Export a table or view to a csv file.  For example, if the `demographic_survey`
+was created in the above section, use this commmand to create a csv::
```

Comment:
what's the CSV output format? do we allow the format to be customizable/easy to use for Excel?